netcdftime/netcdftime.py

```diff
@@ -119,6 +120,12 @@ def JulianDayFromDate(date, calendar='standard'):
     # adjust for Julian calendar if necessary
     jd = jd + B
 
+    if domc:
+        # MC - Add a small offset (proportional to Julian date) for correct re-conversion.
+        eps = 2.22e-016 # ~ Fortran epsilon(1.0) in double precision
```

Comment:
Dear Jeff,

this is true.

I have only read the following in the numpy docu:
For developers of NumPy: do not instantiate this at the module level. The initial calculation of these parameters is expensive and negatively impacts import times. These objects are cached, so calling finfo() repeatedly inside your functions is not a problem.

So it looked to me that just writing the number might be easier.

Regards
Matthias


On 28 Jun 2015, at 13:50, Jeff Whitaker <notifications@github.com> wrote:

> In netcdftime/netcdftime.py:
> 
> > @@ -119,6 +120,12 @@ def JulianDayFromDate(date, calendar='standard'):
> >      # adjust for Julian calendar if necessary
> >      jd = jd + B
> >  
> > +    if domc:
> > +        # MC - Add a small offset (proportional to Julian date) for correct re-conversion.
> > +        eps = 2.22e-016 # ~ Fortran epsilon(1.0) in double precision
> Could use numpy finfo here:
> 
> print(np.finfo(float).eps)
> 2.22044604925e-16
> —
> Reply to this email directly or view it on GitHub.
>